ccd-definition/AuthorisationCaseField/CareSupervision/la-solicitor.json

```diff
@@ -2499,5 +2513,19 @@
     "UserRole": "[LASOLICITOR]",
     "CaseFieldID": "refusedHearingOrders",
     "CRUD": "CRU"
+  },
+  {
```

Comment:
Why's this permission split out like this?

Reply:
Should LASOLICITOR be CRUD?

Reply:
This is aligning the authorisation settings of confidentialRespondents, confidentialOthers and confidentialChildren, which affect the visibility of "Confidential Information" tab.

With this separated setting, I am thinking of one possible scenario where a LABARRISTER updates the respondent as confidential, But only "caseworker-publiclaw-solicitor" with "LASOLICITOR" can view the confidential information in the tab.

Reply:
ok, but why does caseworker-publiclaw-solicitor need CUD, do they need to be able to update it?

Reply:
Think this is ok ... so we're allowing other solicitors to upload a document as confidential